### PR TITLE
Add token counts for LST/LRT

### PR DESCRIPTION
**Pull request by vincenthongzy**
[RESTAKE-178](https://www.notion.so/nethermind/Missing-LST-and-LRT-count-at-the-top-of-their-respective-pages-cbd126026cc74132b6ea82a73b7522d4?pvs=4)

### CODE REVIEW

**Comment by vincenthongzy** [comment]
> Screenshot 2024-08-26 at 2 34 43 PM
> This looks a bit misaligned 🤔

Whoops typo

**Review by Zishan-7 — APPROVED** [review]

File: src/lrt/LRT.jsx [hunk 1]
```diff
@@ -74,3 +74,19 @@ export default function LRT() {
     </>
   );
 }
+
+const getNumberOfLRT = data => {
+  console.log('har', data);
```
Comment (rubo) [inline src/lrt/LRT.jsx:79]: You forgot to remove this:

```suggestion
```

File: src/lrt/LRT.jsx [hunk 2]
```diff
@@ -74,3 +74,19 @@ export default function LRT() {
...
+  }
+
+  const latest = data.length - 1;
+
+  if (!data[latest].protocols) {
+    return 0;
+  }
+
+  // converted by transformProtocols, is now an object
+  return Object.keys(data[latest].protocols).length;
```
Comment (rubo) [inline src/lrt/LRT.jsx:91]: Micro optimization:

```suggestion
  const latest = data[data.length - 1];

  if (!latest.protocols) {
    return 0;
  }

  // converted by transformProtocols, is now an object
  return Object.keys(latest.protocols).length;
```

File: src/lst/LST.jsx [hunk 3]
```diff
@@ -100,3 +103,17 @@ export default function LST() {
...
+    return 0;
+  }
+
+  const latest = data.length - 1;
+
+  if (!data[latest].strategies) {
+    return 0;
+  }
+
+  return data[latest].strategies.length;
```
Comment (rubo) [inline src/lst/LST.jsx:118]: Micro optimization:

```suggestion
  const latest = data[data.length - 1];

  return latest?.strategies?.length ?? 0;
```

**Review by rubo — CHANGES_REQUESTED** [review]

**Review by rubo — APPROVED** [review]

**Pull request merged**